.babelrc

```diff
@@ -1,4 +1,10 @@
 {
-  "presets": [ "es2015-node4" ],
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": "8.7"
```

Comment:
8.9? / maybe change to `"node": "current"` to compile via the node version that runs the babel process?

Reply:
no - that means i can get caught by upgrades without actually wanting to drop the babelification